pkgs/by-name/vu/vulkan-memory-allocator/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vulkan-memory-allocator";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "GPUOpen-LibrariesAndSDKs";
+    repo = "VulkanMemoryAllocator";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-j0Z9OEwQx3RB2cni9eK3gYfwkhOc2ST213b6VseaVzg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
```

Comment:
Is there any difference whether explicitly setting `strictdDeps = true` in cases like these?

Reply:
The output is identical, except for embedded out paths. Should I turn it on? Also could you point me to the documentation for this attribute?

Reply:
https://ryantm.github.io/nixpkgs/stdenv/stdenv/

> Dependencies can be broken down along three axes: their host and target platforms relative to the new derivation’s, and whether they are propagated. The platform distinctions are motivated by cross compilation; see [Cross-compilation](https://ryantm.github.io/nixpkgs/stdenv/cross-compilation/#chap-cross) for exactly what each platform means. [^footnote-stdenv-ignored-build-platform] But even if one is not cross compiling, the platforms imply whether or not the dependency is needed at run-time or build-time, a concept that makes perfect sense outside of cross compilation. By default, the run-time/build-time distinction is just a hint for mental clarity, but with strictDeps set it is mostly enforced even in the native case.

Reply:
Since strictDeps turns on extra checks, I prefer to enable it.

In case of not working, I set it explicitly as `false`, plus a comment summarizing the error.